phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -3497,6 +3516,10 @@ public void addColumn(RpcController controller, final AddColumnRequest request,
                     request.getClientVersion(), parentTable, transformingNewTable, addingColumns);
             if (result != null) {
                 done.run(MetaDataMutationResult.toProto(result));
+
+                metricsSource.incrementAlterAddColumnCount();
+                LOGGER.info("Column(s) added successfully, tableName: {}",
```

Comment:
I'm thinking about logging the names of added columns, but did not found where to get them.